src/app/backing/BackingPage.tsx

```diff
@@ -16,16 +19,25 @@ export const BackingPage = () => {
       data-testid={NAME}
       className="flex flex-col items-start w-full h-full pt-[0.13rem] gap-10 rounded-sm"
     >
-      <PageTitleContainer leftText={NAME}>
-        {/* TODO: ADD CHILDREN HERE OR TEXT IN LEFT_TEXT */}
-      </PageTitleContainer>
-      <div data-testid="CenterContainer" className="flex w-full items-start gap-2">
-        <InfoContainer className="grow-[9]">{/* TODO: ADD CHILDREN HERE */}</InfoContainer>
-        <MetricsContainer className="grow-[3]">{/* TODO: ADD CHILDREN HERE */}</MetricsContainer>
+      <PageTitleContainer leftText={NAME} />
+      <div data-testid="CenterContainer" className="flex w-full items-stretch gap-2">
+        <InfoContainer
+          className="grow-[9] h-full"
+          title="Support innovative Builders by allocating your stRIF to those you align with."
+        >
+          <BackingBanner />
+        </InfoContainer>
+        <MetricsContainer className="grow-[3] h-full">
+          <AnnualBackersIncentivesMetric />
+          <EstimatedRewardsMetric />
+        </MetricsContainer>
       </div>
 
-      {address && <ActionMetricsContainer>{/* TODO: ADD CHILDREN HERE */}</ActionMetricsContainer>}
-      <ActionsContainer title="TODO: ADD TITLE COMPONENT" />
+      {/* {address && <ActionMetricsContainer>{/* TODO: ADD CHILDREN HERE */}
+      {/* </ActionMetricsContainer>} */}
+      <ActionsContainer title="IN THE SPOTLIGHT">
+        <BuildersContainer />
+      </ActionsContainer>
     </div>
```

Comment:
This is not your fault, but, could you please wrap all the containers except for the title container in a flex-col with gap-2, please?

```suggestion
<div data-testid="CenterContainer" className="flex flex-col w-full items-stretch gap-2">
        <div data-testid="CenterContainer" className="flex w-full items-stretch gap-2">
          <InfoContainer
            className="grow-[9] h-full"
            title="Support innovative Builders by allocating your stRIF to those you align with."
          >
            <BackingBanner />
          </InfoContainer>
          <MetricsContainer className="grow-[3] h-full">
            <AnnualBackersIncentivesMetric />
            <EstimatedRewardsMetric />
          </MetricsContainer>
        </div>

        {/* {address && <ActionMetricsContainer>{/* TODO: ADD CHILDREN HERE */}
        {/* </ActionMetricsContainer>} */}
        <ActionsContainer title="IN THE SPOTLIGHT">
          <BuildersContainer />
        </ActionsContainer>
      </div>
```